www/js/diary/infinite_scroll_list.js

```diff
@@ -611,27 +611,89 @@ angular.module('emission.main.diary.infscrolllist',['ui-leaflet',
         nextText: $translate.instant('tour-next'),
         finishText: $translate.instant('tour-finish')
       },
-      steps: []
+      steps: [{
+        target: '.ion-view-background',
+        content: $translate.instant('new_label_tour.0')
+      },
+      {
+        target: '.labelfilter',
+        content: $translate.instant('new_label_tour.1')
+      },
+      {
+        target: '.labelfilter.last',
+        content: $translate.instant('new_label_tour.2')
+      },
+      {
+        target: '.diary-entry',
+        content: $translate.instant('new_label_tour.3')
+      },
+      {
+        target: '.control-icon-button',
+        content: $translate.instant('new_label_tour.4'),
+        before: function() {
+          var d = $q.defer();
+          $ionicScrollDelegate.scrollTop(true);
+          d.resolve();
+          return d.promise
```

Comment:
I think this is the replacement

```suggestion
          return new Promise(function(resolve, reject)  {
             $ionicScrollDelegate.scrollTop(true);
             resolve();
          });
```